### ISSUE TITLE
touchscreen is not working

### ISSUE THREAD

**Issue by lukastillmann** [panbachi/dashzero#4]
I get this error when trying to use the touch screen, which doesn't react to inputs:

`INFO: The key you just pressed is not recognized by SDL. To help get this fixed, please report this to the SDL forums/mailing list <https://discourse.libsdl.org/> EVDEV KeyCode 330 `

@panbachi  I don't know wether you read this, but have you maybe encountered this error before?

**Comment by napsio** [comment]
Yes, I'm still waiting for the Release of the new SHPI OS. I had the same issue with the newer SHPI firmware.

**Comment by phbasler** [comment]
@napsio Is there any news on this? I have libsdl2-dev v 2.0.9 on buster
